src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40 * days;
```

Comment:
It is a good practice to use meaningful variable names. Consider renaming 'rent' to 'totalCost'.